preprocess.py

```diff
@@ -2,15 +2,34 @@
 import numpy as np
 import pickle
 import scipy.ndimage.filters as filters
-
-import BVH as BVH
-import Animation as Animation
-from Quaternions import Quaternions
-from InverseKinematics import BasicJacobianIK, JacobianInverseKinematics
-from Pivots import Pivots
+from absl import app
+from absl import flags
+
+import motion.BVH as BVH
```

Comment:
You can get rid of `motion.` from everywhere: 
1. go to settings> python interpreter > source folders > add the motion folder as one of the source folders if you're using anaconda
2. you have to use the run button provided in the IDE to run from there on you can't use the terminal to run the program

Reply:
I am not using anaconda right now, that's why I cannot run this in my local. Can you make this change and I will set up anaconda in my local

Reply:
I already made these changes, you need to pull from the master :)